provision.sh

```diff
@@ -25,14 +25,12 @@ sudo sysctl -p /etc/sysctl.d/99-ipforward.conf
 # Configure libvirt to accept TCP connections
 # https://github.com/openshift/installer/tree/master/docs/dev/libvirt#configure-libvirt-to-accept-tcp-connections
 sudo bash -c 'cat >> /etc/libvirt/libvirtd.conf' << EOF
-listen_tls = 0
-listen_tcp = 1
 auth_tcp="none"
-tcp_port = "16509"
-EOF
-sudo bash -c 'cat >> /etc/sysconfig/libvirtd' << EOF
-LIBVIRTD_ARGS="--listen"
 EOF
+
+sudo systemctl enable libvirtd-tcp.socket
+sudo systemctl start libvirtd-tcp.socket
+
```

Comment:
no restart of `libvirtd.service` if it's already running? Or are we sure it's not going to be running when `provision.sh` runs?